packages/core/test/toast/toastTests.tsx

```diff
@@ -9,45 +9,55 @@ import { assert } from "chai";
 import { mount, shallow } from "enzyme";
 import * as React from "react";
 
-import { Button, Toast } from "../../src/index";
+import { AnchorButton, Button, Toast } from "../../src/index";
 
 describe("<Toast>", () => {
     it("renders only dismiss button by default", () => {
-        const button = shallow(<Toast message="Hello World" />).find(Button);
-        assert.lengthOf(button, 1);
-        assert.strictEqual(button.prop("iconName"), "cross");
+        const { action, dismiss } = wrap(<Toast message="Hello World" />);
+        assert.lengthOf(action, 0);
+        assert.lengthOf(dismiss, 1);
+        assert.strictEqual(dismiss.prop("iconName"), "cross");
     });
 
     it("clicking dismiss button triggers onDismiss callback with `false`", () => {
         const handleDismiss = sinon.spy();
-        shallow(<Toast message="Hello" onDismiss={handleDismiss} />)
-            .find(Button).simulate("click");
+        wrap(<Toast message="Hello" onDismiss={handleDismiss} />)
+            .dismiss.simulate("click");
         assert.isTrue(handleDismiss.calledOnce, "onDismiss not called once");
         assert.isTrue(handleDismiss.calledWith(false), "onDismiss not called with false");
     });
 
     it("renders action button when action string prop provided", () => {
         // pluralize cuz now there are two buttons
-        const buttons = shallow(<Toast action={{ text: "Undo" }} message="hello world" />).find(Button);
-        assert.lengthOf(buttons, 2);
-        assert.equal(buttons.first().prop("text"), "Undo");
+        const { action } = wrap(<Toast action={{ text: "Undo" }} message="hello world" />);
+        assert.lengthOf(action, 1);
+        assert.equal(action.prop("text"), "Undo");
     });
 
     it("clicking action button triggers onClick callback", () => {
         const onClick = sinon.spy();
-        shallow(<Toast action={{ onClick, text: "Undo" }} message="Hello" />)
-            .find(Button).first().simulate("click");
+        wrap(<Toast action={{ onClick, text: "Undo" }} message="Hello" />)
+            .action.simulate("click");
         assert.isTrue(onClick.calledOnce, "action onClick not called once");
     });
 
     it("clicking action button also triggers onDismiss callback with `false`", () => {
         const handleDismiss = sinon.spy();
-        shallow(<Toast message="Hello" onDismiss={handleDismiss} />)
-            .find(Button).first().simulate("click");
+        wrap(<Toast action={{ text: "Undo" }} message="Hello" onDismiss={handleDismiss} />)
+            .action.simulate("click");
         assert.isTrue(handleDismiss.calledOnce, "onDismiss not called once");
         assert.isTrue(handleDismiss.calledWith(false), "onDismiss not called with false");
     });
 
+    function wrap(toast: JSX.Element) {
+        const root = shallow(toast);
+        return {
+            action: root.find(AnchorButton),
```

Comment:
class isn't as easy as you'd hope. with shallow mounting you need the exact string, or a custom predicate. with full mounting, querying for `".pt-button"` gives you the HTML element, not the React component, so you can't do React-y things to it. 😢